packages/grid/x-data-grid/src/tests/quickFiltering.DataGrid.test.tsx

```diff
@@ -85,6 +86,60 @@ describe('<DataGrid /> - Quick Filter', () => {
         quickFilterLogicOperator: 'and',
       });
     });
+
+    it('should no prettify user input', () => {
+      const { container } = render(<TestCase />);
+
+      fireEvent.change(container.querySelector('input[type=search]'), {
+        target: { value: 'adidas   nike' },
+      });
+      clock.runToLast();
+
+      expect(container.querySelector('input[type=search]').value).to.equal('adidas   nike');
```

Comment:
Better to query the `role`.

```suggestion
      fireEvent.change(screen.getByRole('searchbox'), {
        target: { value: 'adidas   nike' },
      });
      clock.runToLast();

      expect(screen.getByRole('searchbox').value).to.equal('adidas   nike');
```

x other tests.